.github/workflows/native_jni_s3_pytorch.yml

```diff
@@ -140,30 +140,26 @@ jobs:
       - name: Instal CUDA
         shell: cmd
         run: |
-          curl.exe -L http://developer.download.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.243_426.00_windows.exe -o cuda.exe
           curl.exe -L http://developer.download.nvidia.com/compute/cuda/10.2/Prod/local_installers/cuda_10.2.89_441.22_windows.exe -o cuda102.exe
           curl.exe -L https://developer.download.nvidia.com/compute/redist/cudnn/v7.6.4/cudnn-10.1-windows7-x64-v7.6.4.38.zip -o cudnn.zip
```

Comment:
@lanking520 is this right cudnn-10.1 ?

Reply:
It should be generally applicable for 10.2, you can try to switch to 10.2